content/en/docs/started/installing-kubeflow.md

```diff
@@ -343,19 +343,16 @@ try out the end-to-end Kubeflow Platform capabilities.
 Users may choose to install the manifests for a specific Kubeflow version by following the
 instructions in the `README` of the [`kubeflow/manifests`](https://github.com/kubeflow/manifests) repository.
 
-- [**Kubeflow 1.8:**](/docs/releases/kubeflow-1.8/)
+- [**Kubeflow 1.8.1:**](/docs/releases/kubeflow-1.8/)
   - [`v1.8-branch`](https://github.com/kubeflow/manifests/tree/v1.8-branch#installation) <sup>(development branch)</sup>
-  - [`v1.8.0`](https://github.com/kubeflow/manifests/tree/v1.8.0#installation)
+  - [`v1.8.1`](https://github.com/kubeflow/manifests/tree/v1.8.1#installation)
 - [**Kubeflow 1.7:**](/docs/releases/kubeflow-1.7/)
   - [`v1.7-branch`](https://github.com/kubeflow/manifests/tree/v1.7-branch#installation) <sup>(development branch)</sup>
   - [`v1.7.0`](https://github.com/kubeflow/manifests/tree/v1.7.0#installation)
 
 {{% alert title="Warning" color="warning" %}}
-Kubeflow is a complex system with many components and dependencies.
-Using the Kubeflow manifests requires a deep understanding of Kubernetes, Istio, and Kubeflow itself.
-
-When using the Kubeflow manifests, the community is not able to provide support for environment-specific issues or custom configurations.
-If you need support, please consider using a [packaged distribution](#packaged-distributions).
+Using the Kubeflow manifests requires some understanding of Kubernetes.
+The Kubeflow community support for Kubeflow manifests is best-effort for environment-specific issues or custom configurations.
```

Comment:
It's important that we hit two points in this warning to avoid users becoming frustrated:

1. The user needs to be familiar with Kubernetes, Istio, and Kubeflow itself to successfully use the manifests for anything more than testing. Saying `"some understanding of Kubernetes"` does not adequately represent this.
2. The Kubeflow community does not provide support for environment-specific issues. 
We don't officially provide `"best effort support"`, it's important that users understand they will be largely supporting themselves, and that they should reach out to a vendor if they need support.

If we want to soften the language without removing these key points, here is my proposal:

```suggestion
Using the Kubeflow Manifests in production requires working knowledge of Kubernetes, Istio, and Kubeflow itself.
When using the Kubeflow manifests, the community is not able to provide support for environment-specific issues or custom configurations. If you need support, please consider using a [packaged distribution](#packaged-distributions).
```

Reply:
Regarding 

1.
I think Basic Kubernetes knowledge is enough. You do not need to understand the details of Istio to get it running for example. Many users install and use it succesfully without understanding Istio. 
"Kubeflow itself" does not make sense to me, because you need to install it first to become familiar with Kubeflow.

2. 
We do provide best effort support on Slack and in the repository/issue tracker, so that is not true.

I can change it to:
```
{{% alert title="Warning" color="warning" %}} Using the Kubeflow manifests requires basic Kubernetes knowledge.
The Kubeflow community support for Kubeflow manifests is only best-effort and not guaranteed for environment-specific issues or custom configurations.  Nevertheless, we welcome contributions and bug reports very much. {{% /alert %}}
```